.github/workflows/test.yml

```diff
@@ -20,5 +20,7 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install pytest
           python -m pip install jsonschema
+      - name: Generate Test SQs
+        run: python sq-generator/sq_generator.py -i test/resources/testdata.json -o test/resources/generated-sqs/
```

Comment:
The file should be called `test-data.json` because `test` and `data` are two individual words.